### ISSUE TITLE
[BUG] Field Username is not accessible

### ISSUE THREAD

**Issue by mochour** [owncloud/ios-app#843]
### Steps to reproduce
1. Buy iPhone 12 Mini
2. Upgrade the phone from iPhone 7 SE
3. Upgrade Owncloud ios-app 
4. Open Owncloud ios-app
6. Try to reconnect to the Owncloud server

### Expected behavior
When I try to reconnect to the server all fields are filled except Username and Password.
I can fill my Username and Password and login. 

### Actual behavior
I can't access field Username and fill a credential.
I can access field Password and fill a password. 

### Client
**iOS version: 14.2.1**

**ownCloud app version: 11.4.3**

**Device model: iPhone 12 Mini**

### Server configuration
**Web server: Apache2 2.4.2**

**Database: MySQL**

**PHP version: 7**

**ownCloud version: 10.5.0.10**

**Storage backend (external storage):**

#### Web server error log
```
Insert your webserver log here
```

#### ownCloud log (data/owncloud.log)
```
Insert your ownCloud log here
```

**Comment by ChrisEdS** [comment]
> 1. Buy iPhone 12 Mini

... made my day! 🤣 

Thank you for reporting this issue. We just ordered the iPhone. :)

**Comment by mochour** [comment]
:-) nice ... the test has to be complete

**Comment by hosy** [comment]
@mochour should be fixed with version 11.4.4.

**Comment by mochour** [comment]
Thanks for support.
